terms.html

```diff
@@ -137,8 +137,9 @@
   <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
   <dd>
 The <a>entity</a> about which a <a>claim</a> is made.
-The common practice of speaking about "the subject of a <a>credential</a>" SHOULD imply
-that such a <a>credential</a> contains precisely one <a>claim</a>.
+The common practice of speaking about "the <a>subject</a> of a <a>credential</a>" SHOULD imply 
+that such a <a>credential</a> contains precisely one or more <a>claims</a> about precisely one <a>entity</a>,
```

Comment:
`precisely` should only be used with some `precise` number(s), not the intentionally vague `one or more`.
```suggestion
that such a <a>credential</a> contains one or more <a>claims</a> about precisely one <a>entity</a>,
```